api/src/domain/user/taxEncryptionDecryptionFactory.ts

```diff
@@ -0,0 +1,50 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const AWSCrypto = require("@aws-crypto/client-node");
+import { fromBase64, toBase64 } from "@aws-sdk/util-base64-browser";
+import { TextDecoder } from "node:util";
+import { EncryptionDecryptionClient } from "../types";
+
+type context = {
+  stage: string;
+  purpose: string;
+  origin: string;
+};
+
+export const TaxEncryptionDecryptionFactory = (
```

Comment:
again, no need for `Tax` to be here. Nothing here is Tax specific.

This implementation is a specific type of an Encryption client - but it's not tax-specific.

What makes this implementation unique? It's that it uses AWS. We _could_ write any encryption client that uses any random encrytion algorithm. This one is unique because it's an AWS Encryption Client - and that's how it should be named